_functions.sh

```diff
@@ -237,6 +237,7 @@ initialize_product_settings() {
       configurable_env_var "DEPLOYMENT_ADDONS" ""
       configurable_env_var "DEPLOYMENT_ADDONS_CATALOG" ""
       configurable_env_var "DEPLOYMENT_ADDONS_MANAGER_CONFLICT_MODE" "" # used for add-on manager --conflict parameter (default: none) (possible values: fail / skip / overwrite)
+      configurable_env_var "DEPLOYMENT_ADDONS_MANAGER_NOCOMPAT_MODE" "" # used for add-on manager --no-compat parameter (default: none) (possible values: true / false)
```

Comment:
You may use the DEPLOYMENT_ADDONS_MANAGER_CONFLICT_MODE, it is already defined for that purpose

Reply:
Ok, the option is not the same.